Vagrantfile

```diff
@@ -253,12 +310,13 @@ def provision(config,
     #{extra}
 
     installed java || {
-      echo "==> Java is not installed on vagrant box ${config.vm.box}"
+      echo "==> Java is not installed"
       return 1
     }
     ensure tar
     ensure curl
     ensure unzip
+    ensure rsync
```

Comment:
rsync is used by the [platform tests](https://github.com/elastic/elasticsearch/blob/caa63ada6bdf7b80fbd2c283aebdd9fe798dde1d/buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy#L47) (which from my understanding we'd like to start using again) and is missing from some of the boxes